src/backend/util2.c

```diff
@@ -54,6 +54,8 @@ void util_assert(const char *file, int line)
 {
     fflush(stdout);
     printf("Internal error: %s %d\n",file,line);
+    printf("This is a compiler bug, please report it via "
+        "https://issues.dlang.org/enter_bug.cgi\n");
```

Comment:
1) How about pre-filling some selections?
e.g. https://issues.dlang.org/enter_bug.cgi?component=dmd&bug_severity=critical&keywords=ice

(we could make a short URL or short domain (e.g. `https://bugs.dlang.io/ice` out of this, if that's too long)

2) As someone on the NG proposed: What about a link to the DWiki instead which describes further steps? (e.g. how to dustmite, what to mention in the bug report, ...)
- Just an empty bug report page can be quite intimidating for a user.
- We don't seem to have such a page as of now, but maybe we can take a bit of information from [Get involved](http://wiki.dlang.org/Get_involved) and start a new one?
- the [issue writing guidelines](https://issues.dlang.org/page.cgi?id=bug-writing.html) seems to be a generic page from Bugzilla